umpost/um2netcdf.py

```diff
@@ -635,6 +602,81 @@ def add_global_history(infile, iris_out):
     warnings.warn("um2nc version number not specified!")
 
 
+# TODO: refactor func sig to take exclusive simple OR unique name field?
+def rename_cube_var_name(cube, um_var, simple: bool):
+    """
+    Modify cube `var_name` attr to change naming for NetCDF output.
+
+    Parameters
+    ----------
+    cube : iris cube to modify (changes the name in place)
+    um_var : the UM Stash code structure
+    simple : True to replace var_name with "fld_s00i000" style name
+    """
+    if simple:
+        stash_code = cube.attributes[STASH]
+        cube.var_name = f"fld_s{stash_code.section:02}i{stash_code.item:03}"
+    elif um_var.uniquename:
+        cube.var_name = um_var.uniquename
+
+    # Could there be cases with both max and min?
+    if cube.var_name:
+        if any([m.method == 'maximum' for m in cube.cell_methods]):
+            cube.var_name += "_max"
+        if any([m.method == 'minimum' for m in cube.cell_methods]):
+            cube.var_name += "_min"
+
+
+def rename_cube_standard_names(cube, um_var, verbose: bool):
+    """
+    Modify cube `standard_name` attr to change naming for NetCDF output.
+
+    Parameters
+    ----------
+    cube : iris cube to modify (changes the name in place)
+    um_var : the UM Stash code structure
+    verbose : True to turn warnings on
+    """
+    stash_code = cube.attributes[STASH]
+
+    # The iris name mapping seems wrong for these - perhaps assuming rotated grids?
+    if cube.standard_name:
+        if cube.standard_name == 'x_wind':
+            cube.standard_name = 'eastward_wind'
+        if cube.standard_name == 'y_wind':
+            cube.standard_name = 'northward_wind'
+
+        if cube.standard_name and um_var.standard_name:
+            if cube.standard_name != um_var.standard_name:
+                # TODO: remove verbose arg & always warn? Control warning visibility at cmd line?
+                if verbose:
+                    msg = (f"Standard name mismatch section={stash_code.section}"
+                           f" item={stash_code.item} standard_name={cube.standard_name}"
+                           f" UM var name={um_var.standard_name}")
+                    warnings.warn(msg)
+
+                cube.standard_name = um_var.standard_name
+    elif um_var.standard_name:
+        # If there's no standard_name or long_name from iris, use one from STASH
```

Comment:
```suggestion
        # If there's no standard_name from iris, use one from STASH
```
Maybe add a similar comment to `rename_cube_long_names()`?

Reply:
That's updated in https://github.com/ACCESS-NRI/um2nc-standalone/pull/52/commits/5be1420434ce79bf6923f0f9a177f645a4e59de4.